
max_iterations : 1000
#  the small number which will terminate loop if path difference smaller than this number.
epsilon : 1.0e-4
#maximum possible curvature of the non-holonomic vehicle
min_turning_radius : 7
#maximum distance to obstacles that is penalized
obsd_max : 4
# maximum distance for obstacles to influence the voronoi field
vor_obs_dmax : 3
#falloff rate for the voronoi field
alpha: 0.1
# weight for the obstacle term
weight_obstacle : 2.0
# weight for the voronoi term
weight_voronoi : 2.0
# weight for the curvature term
weight_curvature : 0.0
# weight for the smoothness term
weight_smoothness : 2.0
#weight for path length
weight_length : 0.1